pkg/cmd/purge.go

```diff
@@ -117,6 +113,52 @@ var purgeCmd = &cobra.Command{
 		ctx = context.WithValue(ctx, telemetry.CLIENT_ID_CONTEXT_KEY, config.GetClientId())
 		ctx = context.WithValue(ctx, telemetry.ENABLED_CONTEXT_KEY, c.TelemetryEnabled)
 
+		// Get all targets, workspaces and builds to prompt user for resource purge
+		targets, err := server.TargetService.ListTargets(ctx, nil, services.TargetRetrievalParams{})
+		if err != nil {
+			if !forceFlag {
+				return err
+			} else {
+				fmt.Printf("Failed to get targets: %v\n", err)
+			}
+		}
+
+		workspaces, err := server.WorkspaceService.ListWorkspaces(ctx, services.WorkspaceRetrievalParams{})
+		if err != nil {
+			if !forceFlag {
+				return err
+			} else {
+				fmt.Printf("Failed to get workspaces: %v\n", err)
+			}
+		}
+
+		builds, err := server.BuildService.List(ctx, nil)
+		if err != nil {
+			if !forceFlag {
+				return err
+			} else {
+				fmt.Printf("Failed to get builds: %v\n", err)
+			}
+		}
+
+		if len(targets) != 0 || len(workspaces) != 0 || len(builds) != 0 {
+			var continuePurge bool
+			commands := view.PurgeResourcesPrompt(&continuePurge, len(targets), len(workspaces), len(builds))
+			if err != nil {
+				if !forceFlag {
+					return err
+				} else {
+					fmt.Printf("Failed to prompt for resource purge: %v\n", err)
+				}
+			}
+			if !continuePurge {
+				fmt.Printf("\nOperation cancelled.\nManually delete leftover resources for a complete purge by running following commands:\n%s\n", strings.Join(commands, "\n"))
```

Comment:
Add one more new line after `by running following commands:`.

Also, I suggest that we remove `Delete {targets|workspaces|builds: ` and just print out the commands. That's easier to copy paste.

Reply:
Also, we should add "by starting the server and running the following commands"